src/realm/sync/noinst/migration_store.cpp

```diff
@@ -124,10 +138,20 @@ std::shared_ptr<realm::SyncConfig> MigrationStore::convert_sync_config(std::shar
             return config;
         }
         if (m_state == MigrationState::NotMigrated) {
+            // PBS config, store the partition value locally in case a migration occurs later
+            m_migrated_partition = config->partition_value;
             return config;
         }
     }
 
+    // Once in the migrated state, the partition value cannot change for the same realm file
```

Comment:
just to confirm - this is not new behavior, right? it would always be invalid upstream for a realm file to be used with different partition values?

Reply:
I don't believe so and this is to verify that it doesn't happen, esp. while the client is in the migrated state.